README.md

```diff
@@ -170,3 +184,19 @@ information.
 ## Security Policy <!-- omit in toc -->
 
 See [SECURITY](SECURITY.md) for more information.
+
+## Notices and Disclaimers
+
+Intel technologies may require enabled hardware, software or service activation.
+
+No product or component can be absolutely secure.
+
+Your costs and results may vary.
+
+© Intel Corporation. Intel, the Intel logo, and other Intel marks are trademarks of Intel Corporation or its subsidiaries. Other names and brands may be claimed as the property of others.
```

Comment:
this covers the trademark issue, is that right ?

Reply:
Yes.